cogs/help.py

```diff
@@ -38,27 +52,85 @@ async def create_view(self):
             self.bot.add_view(HelpView())
 
     @commands.Cog.listener()
-    async def on_message(self, message):
+    async def on_message(self, message: Message):
         if message.channel.id == self.help_channel and message.type == MessageType.thread_created:
             await message.delete(delay=5)
 
-    @commands.command()
+    @commands.command(hidden=True)
     @commands.is_owner()
-    async def help_menu(self, ctx):
+    async def help_menu(self, ctx: commands.Context):
         await ctx.send("Click a button to create a help thread!", view=HelpView())
 
-    @commands.command()
-    async def close(self, ctx):
+    @commands.command(name="Close", aliases=['cs'])
+    async def close(self, ctx: commands.Context):
         history = ctx.channel.history(oldest_first=True, limit=1)
-        history_flat = await history.flatten()
-        log_channel = nextcord.utils.get(ctx.guild.channels, name='help_logs')
-        help_role = nextcord.utils.find(lambda r: r.name == 'My help is bad', ctx.message.guild.roles)
-        if help_role in ctx.author.roles or history_flat[0].mentions[0].id == ctx.author.id:
-            if isinstance(ctx.channel, nextcord.Thread) and ctx.channel.parent_id == self.help_channel:
-                await ctx.send("This thread has now been closed. Please create another thread if you wish to ask another question.")
+        if isinstance(ctx.channel, Thread) and ctx.channel.parent_id == self.help_channel:
+            history_flat = await history.flatten()
+            log_channel: TextChannel = get(ctx.guild.channels, name='help_logs')
+            help_role: Role = find(lambda role: role.name == 'My help is bad', ctx.message.guild.roles)
+            if help_role in ctx.author.roles or history_flat[0].mentions[0].id == ctx.author.id:
+                await ctx.send(
+                    "This thread has now been closed. Please create another thread if you wish to ask another question.")
                 await ctx.channel.edit(locked=True, archived=True)
-                await log_channel.send(f"Help thread {ctx.channel.name} (created by {history_flat[0].mentions[0].name}) has been closed.")
+                await log_channel.send(
+                    f"Help thread {ctx.channel.name} (created by {history_flat[0].mentions[0].name}) has been closed.")
+
+    @commands.command(name='Help', aliases=['hp'],
+             help="Displays the commands of me.",
+             usage="help|hp (command|command alias|category|category alias)")
+    async def help(self, ctx: Context, *, arg: Optional[str]):
```

Comment:
Optional typehints doesnt work with no defaults.

Reply:
Oh, sorry ill keep that in mind,

Reply:
you seem to have marked a lot as resolved and not done anything